server/constants/vars.js

```diff
@@ -62,3 +64,49 @@ export const ALL_SUBJECTS_MONGO_PROJECTION = {
   _id: 0,
   subject: 1,
 }
+
+export const DEFAULT_CONFIG_AT_SIGNUP = (uid) => ({
+  owner: uid,
+  config: defaultUserConfig,
+  name: 'default',
+  type: 'matrix',
+  readers: [uid],
+  created: new Date().toUTCString(),
+})
+
+export const SIGN_IN_MONGO_PROJECTION = {
```

Comment:
should this be something more generic like `USER_PROJECTION`? Is this the projection we should be using for users even when not singing in?

```suggestion
export const USER_PROJECTION = {
```

Reply:
[de48040](https://github.com/AMP-SCZ/dpdash/pull/293/commits/de48040cb5b39f723d4ad1877be815b064027992)